src/Common/Sanitizer/IOutputSanitizer.cs

```diff
@@ -12,12 +12,18 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using System.Collections.Generic;
+
 namespace Microsoft.WindowsAzure.Commands.Common.Sanitizer
 {
     public interface IOutputSanitizer
     {
         bool RequireSecretsDetection { get; }
 
+        IEnumerable<string> IgnoredModules { get; }
```

Comment:
why IEnumerable not HashSet?

Reply:
Just want to keep the interface definition more flexible and enumeration is the necessary feature of the collection. In the implementation, I will use string array instead as IgnoredModules and IgnoredCmdlets are both small lists while HashSet is efficient for large list and looks like a bit heavy for this scenario.